src/Surfnet/StepupMiddleware/ApiBundle/Resources/config/routing.yml

```diff
@@ -38,6 +38,12 @@ verified_second_factors:
     methods:  [GET]
     condition: "request.headers.get('Accept') matches '/^application\\\\/json($|[;,])/'"
 
+verified_second_factors_no_authz:
+    path:     /self-service-verified-second-factors
```

Comment:
Please rename to be more descriptive from the viewpoint of SS and current identity.

Reply:
Yes, renamed to `verified-second-factors-of-identity`